dayrepo/snippets/templates/snippet_list.html

```diff
@@ -14,6 +17,12 @@ <h2>提出済み日報一覧</h2>
     <button onclick="location.href='{% url 'excelfile_download' post.id %}'">出力</button>
     <button onclick="location.href='{% url 'checklist_edit' post.checklist_id.id %}'">点検編集</button>
     <button onclick="location.href='{% url 'snippet_edit' post.id %}'">日報編集</button>
+    <form method="post" action={% url 'db_delete' post.id 'snippet' %}>
+        {% csrf_token %}
+        <button type="submit">日報削除</button>
+    <form method="post" action={% url 'db_delete' post.id 'all' %}>
+        {% csrf_token %}
+        <button type="submit">一括削除</button>
```

Comment:
form タグ閉じてなくないです？
バグの元なので気をつけましょう〜

```suggestion
    <form method="post" action={% url 'db_delete' post.id 'snippet' %}>
        {% csrf_token %}
        <button type="submit">日報削除</button>
    </form>
    <form method="post" action={% url 'db_delete' post.id 'all' %}>
        {% csrf_token %}
        <button type="submit">一括削除</button>
    </form>
```